blocks/api/registration.js

```diff
@@ -44,6 +49,18 @@ export function registerBlockType( name, settings ) {
 		);
 		return;
 	}
+	if ( ! settings || ( settings && ! isFunction( settings.save ) ) ) {
```

Comment:
The `settings &&` check here is redundant - by this point in the conditional, we already know that `settings` is set.